### ISSUE TITLE
suggestion - effect listed in item description

### ISSUE THREAD

**Issue by definitely-nobody-is-here** [maitian352/Meadow-Guarder#283]
mystic knight helmet does not say it shoots arrows that freeze monsters but it does. So I have to make one to find out what it does and it makes it hard to get a good setup without wasting tons of time.

**Comment by Suvanth-Erranki** [comment]
I agree, but I also want to add that crafting should list how much damage projectiles from an item do (even if it is not a weapon), because it can differ from item to item.
